include/nbl/builtin/glsl/ext/CentralLimitBoxBlur/parameters.glsl

```diff
@@ -0,0 +1,64 @@
+#ifndef _NBL_GLSL_EXT_BLUR_PARAMETERS_INCLUDED_
+#define _NBL_GLSL_EXT_BLUR_PARAMETERS_INCLUDED_
+
+#include "nbl/builtin/glsl/ext/CentralLimitBoxBlur/parameters_struct.glsl"
+
+#ifndef _NBL_GLSL_EXT_BLUR_GET_PARAMETERS_DECLARED_
+nbl_glsl_ext_Blur_Parameters_t nbl_glsl_ext_Blur_getParameters();
+#define _NBL_GLSL_EXT_BLUR_GET_PARAMETERS_DECLARED_
+#endif
```

Comment:
be consitent with naming, the macros/defines should have `EXT_CENTRAL_LIMIT_BOX_BLUR` instead of `EXT_BLUR`

the `nbl_glsl_ext_Blur` should be `nbl_glsl_ext_CentralLimitBoxBlur` as well